src/olympia/files/models.py

```diff
@@ -706,6 +712,50 @@ class Meta:
         db_table = 'validation_annotations'
 
 
+class WebextPermission(ModelBase):
+    KNOWN_PERMISSIONS = WEBEXT_PERMISSIONS_DICT
+    name = models.CharField(max_length=255, unique=True, null=False)
+    _enum = models.IntegerField(default=WEBEXT_UNKNOWN.id, db_column='enum')
+    files = models.ManyToManyField(File, related_name='webext_permissions',
+                                   through='FileWebextPermission')
+
+    class Meta(ModelBase.Meta):
+        db_table = 'webext_permissions'
+
+    def __init__(self, *args, **kw):
+        super(WebextPermission, self).__init__(*args, **kw)
+
+        enum = WEBEXT_PERMISSIONS_DICT.get(self.name, WEBEXT_UNKNOWN).id
+        self.update(_enum=enum)
```

Comment:
We don't want to trigger a database save each time we instantiate this class... Another strike against having `enum` for me